crates/sui-adapter/src/adapter.rs

```diff
@@ -371,8 +374,11 @@ pub fn generate_package_id(
     let package_id = ctx.fresh_id();
     for module in modules.iter() {
         let old_module_id = module.self_id();
+        let package_name = old_module_id.name().as_str();
+        let is_framework =
+            package_name == SUI_PACKAGE_NAME || package_name == MOVE_STDLIB_PACKAGE_NAME;
         let old_address = *old_module_id.address();
-        if old_address != AccountAddress::ZERO {
+        if !is_framework && old_address != AccountAddress::ZERO {
```

Comment:
What's the reason behind this change?

Reply:
It was a refactor, but I've reverted it since it is not related or neccessary.